src/main/java/WayofTime/bloodmagic/item/ItemBloodOrbTesting.java

```diff
@@ -0,0 +1,73 @@
+package WayofTime.bloodmagic.item;
+
+import WayofTime.bloodmagic.BloodMagic;
+import WayofTime.bloodmagic.client.IVariantProvider;
+import WayofTime.bloodmagic.core.data.SoulNetwork;
+import WayofTime.bloodmagic.util.helper.NetworkHelper;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraft.util.text.TextComponentTranslation;
+import net.minecraft.world.World;
+
+public class ItemBloodOrbTesting extends Item implements IVariantProvider {
+
+    public ItemBloodOrbTesting() {
+        setTranslationKey(BloodMagic.MODID + ".orb.testing");
+        setCreativeTab(BloodMagic.TAB_BM);
+        setMaxStackSize(1);
+    }
+
+    @Override
+    public ActionResult<ItemStack> onItemRightClick(World world, EntityPlayer player, EnumHand hand) {
+        SoulNetwork network = NetworkHelper.getSoulNetwork(player);
+        if (!world.isRemote)
+            if (player.isSneaking()) {
+                network.setCurrentEssence(Integer.MAX_VALUE);
+                player.sendMessage(new TextComponentTranslation("commands.bloodmagic.network.fill.success", player.getDisplayName().getFormattedText()));
```

Comment:
`TextComponentTranslation` allows `ITextComponents` to be passed to it, so the user's name should remain as an `ITextComponent`.